src/PhpPact/Consumer/Matcher/Exception/InvalidUuidFormatException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace PhpPact\Consumer\Matcher\Exception;
+
+use Exception;
+
+class InvalidUuidFormatException extends Exception
```

Comment:
Interesting, perhaps slightly off-topic observation.

I'd likely have the MatcherException live under `PhpPact\Consumer\Exceptions` and extend something like `PhpPact\Consumer\Exceptions\ConsumerException`, which itself would extend `PhpPact\Exceptions\BaseException`.

Just the naked word `Exception` is a confusing alias for a human reviewer (I'd imagine some static analysis tools may even panic at it's name shadowing a built-in).

Reply:
Added these exceptions in https://github.com/pact-foundation/pact-php/pull/382